app/uk/gov/hmrc/channelpreferences/model/ChannelPreferencesError.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.channelpreferences.model
+
+sealed trait ChannelPreferencesError
+
+final case class PreferencesConnectorError(message: String) extends ChannelPreferencesError
+final case class UnExpectedError() extends Throwable
```

Comment:
just a tiny typo, it should be UnexpectedError as opposed to UnExpectedError

Reply:
I will change it thanks